d-rita/fast-food-fast-react
# #164353023 Users should be able to log in

#### What does this PR do?
This PR enables a signed up user to log in to the application. 

#### Description of tasks to be completed
- Add login view
- Add login action
- Connect login actions to auth reducer
- Create login form
- Add login route

#### How to test this?
- Fetch and check out this branch using `git fetch origin ft-login-164353023 && git checkout ft-login-164353023`
- Install dependencies using `yarn install`
- Start the server using `yarn start`
- Go to `localhost:8080/login` and sign in a signed up user. 

#### Related Pivotal Tracker Story 
[#164353023](https://www.pivotaltracker.com/story/show/164353023)
